PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalClient.kt

```diff
@@ -147,6 +145,18 @@ class PayPalClient internal constructor(
             .metadata(metadata)
     }
 
+    @Throws(JSONException::class)
+    private fun buildBrowserSwitchOptionsForEditFI(
+        approvalUrl: String
+    ): BrowserSwitchOptions {
+
+        return BrowserSwitchOptions()
+            .requestCode(BraintreeRequestCodes.PAYPAL.code)
```

Comment:
I can't remember if the request codes are still used anymore, but in case they are this should probably have it's own request code (separate from the regular PayPal flow)

Reply:
Looks like the same [request codes](https://github.com/braintree/braintree_android/blob/main/BraintreeCore/src/main/java/com/braintreepayments/api/core/BraintreeRequestCodes.kt) are used for each feature client. cc: @jaxdesmarais